docs/reference/services.rst

```diff
@@ -4,10 +4,64 @@
 MicroOVN services
 =================
 
+MicroOVN has lots of its functionality separated into distinct services that can
+be controlled via MicroOVN enable/disable or snap/systemd, only the former is
+recommended.
+
 This page presents a list of all MicroOVN services. Their descriptions are
 for reference only - the user is not expected to interact directly with these
 services.
 
+Handling services with enable/disable
+-------------------------------------
+
+The status of all services is displayed by running:
+
+.. code-block:: none
+
+   microovn status
+
+``central service``
+-------------------
+
+This is responsible for the database control. The database is a RAFT database so
+it can handle (n-1)/2 failures, where n is the number of nodes.
```

Comment:
Would it make sense to add a link to the [OVSDB clustered database service model](https://docs.openvswitch.org/en/latest/ref/ovsdb.7/#clustered-database-service-model) documentation?